lib/Conversion/TorchToTcp/Utils.h

```diff
@@ -22,5 +22,21 @@ Value broadcastShapeInLeadingDims(ConversionPatternRewriter &rewriter,
 Value broadcastInLeadingDimsToMatchShape(ConversionPatternRewriter &rewriter,
                                          Value input, Value target);
 
+Value broadcast0DToNDAndMatchShape(ConversionPatternRewriter &rewriter,
+                                   Value input, Value target);
+
+Value broadcast1DToNDAndMatchShape(ConversionPatternRewriter &rewriter,
+                                   Value input, Value target,
+                                   int64_t axisInOutput);
```

Comment:
These and earlier functions should have comments (maybe just one combined comment on the top).